bin/install_perl_modules

```diff
@@ -16,11 +16,11 @@ else
     $DIR/vendor/bin/carton install --deployment --without uk --without zurich --without kiitc
 fi
 
-if ! perl -MImage::Magick -e 'exit()' >/dev/null 2>&1
+if ! bin/cron-wrapper perl -MImage::Magick -e 'exit()' >/dev/null 2>&1
```

Comment:
What does bin/cron-wrapper do exactly?

Reply:
It sets up the local::lib for letting perl scripts find modules installed in `local`.